src/ioctl.vapi

```diff
@@ -46,13 +46,13 @@ namespace Ioctl {
     [CCode (cheader_filename = "linux/usbdevice_fs.h")]
     public const int USBDEVFS_CAP_SUSPEND;
 
-    [CCode (cheader_filename = "asm-generic/ioctl.h")]
+    [CCode (cheader_filename = "asm/ioctl.h")]
```

Comment:
This commit is already in master, I suppose that's what it complains about.

Reply:
Oopsa, no, I forgot to rebase!